src/app.css

```
@@ -80,6 +83,12 @@
 		color: green;
 		@apply bg-[#9f9f9f];
 	}
+
+	.tei-gap3.fragment-loss {
```

ich würde vorschlagen die Regel etwas permissiver zu machen:

```suggestion
	.fragment-loss {
```

über die verteilung der Klasse _tei-gap3_ haben wir keinen Einfluss. Es könnte sein, dass wenn das ODD das nächste Mal verändert wird, plötzlich die Klasse _tei-gap2_ oder irgendwas anderes gesetzt wird. `.fragment-loss` setzen wir bewusst im ODD